platform/platform-configuration-service/src/main/java/org/codice/ddf/configuration/service/PlatformUiConfigurationService.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.service;
+
+public interface PlatformUiConfigurationService {
+
+  String getConfig();
```

Comment:
✏️ This method is difficult to understand without going to the impl. How is the returned String formatted? I'd recommend either 1) Adding javadoc or 2) Renaming this method to something like `getConfigAsJsonString`. Or maybe both.